app/models/user.rb

```diff
@@ -51,6 +51,7 @@ class User < ApplicationRecord
   has_many :webhook_endpoints, class_name: "Webhook::Endpoint", foreign_key: :user_id, inverse_of: :user, dependent: :delete_all
   has_many :user_blocks
   has_one :pro_membership, dependent: :destroy
+  has_many :created_podcasts, class_name: "Podcast", foreign_key: :creator_id, inverse_of: :creator, dependent: :nullify
```

Comment:
I was going to ask "why not just .podcasts" but I can see why in the future we might want to have podcast editors or other podcasts tied to a users in different capacity